astropy/coordinates/earth.py

```diff
@@ -699,6 +721,29 @@ def get_gcrs_posvel(self, obstime):
         obsgeovel = gcrs_data.differentials['s'].to_cartesian()
         return obsgeopos, obsgeovel
 
+    def get_cirs_posvel(self, obstime):
```

Comment:
If we do the above, this would be  small function inside the transformation code; a further advantage is that if we move eventually to a new `observer` argument (or gain the ability to store velocity on `location`), we have not introduced a method with relatively little use.